drivers/hud_apr/app/models/hud_apr/generators/shared/fy2024/question_sixteen.rb

```diff
@@ -67,7 +67,17 @@ def self.table_descriptions
     end
 
     private def income_levels(suffix)
-      not_collected = a_t["income_total_at_#{suffix}".to_sym].eq(nil)
+      not_collected = a_t["income_from_any_source_at_#{suffix}".to_sym].eq(nil).
+        or(a_t["income_from_any_source_at_#{suffix}".to_sym].eq(99)).
+        or(
+          a_t["income_total_at_#{suffix}"].eq(nil).
+          and(
+            # Responses of 8 & 9 are expected to have total income nil.
+            # These are filtered out to prevent duplicates. They are captured in a different row.
+            a_t["income_from_any_source_at_#{suffix}"].not_eq(8).
+            and(a_t["income_from_any_source_at_#{suffix}"].not_eq(9)),
```

Comment:
same comment as above if that works as expected.

Reply:
This has been updated in [this commit](https://github.com/greenriver/hmis-warehouse/pull/5000/commits/6d299d090919f6b3640dbb18b822a4c0eae6593c)